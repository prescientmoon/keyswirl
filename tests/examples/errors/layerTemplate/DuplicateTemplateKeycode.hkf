module exporting ()

template foo: a b a -- a appears twice here
