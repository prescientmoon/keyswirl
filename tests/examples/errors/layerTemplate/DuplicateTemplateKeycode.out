[error DuplicateTemplateKeycode]: Keycode "a" appears in template more than once
     +--> errors.layerTemplate.DuplicateTemplateKeycode@3:19-3:20
     |
   3 | template foo: a b a -- a appears twice here
     :          ^--  ^   ^ 
     :          |    |   `- the second occurence of this keycode
     :          |    `- the first occurence of this keycode
     :          `- while checking this layer template
     :
     | Hint: Try removing all but one of the highlighted occurences
-----+
